test/JDBC/expected/string_agg-vu-verify.out

```diff
@@ -0,0 +1,560 @@
+GO
+~~START~~
+text
+ht-dw-cx-az
+ev-gu-by
+~~END~~
+
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+ht-dw-cx-az
+ev-gu-by
+~~END~~
+
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+az-cx-dw-ht
+by-gu-ev
+~~END~~
+
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+az-cx-ht-dw
+gu-by-ev
+~~END~~
+
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+az-cx-dw-ht
+by-gu-ev
+~~END~~
+
+
+-- Delimeter as a function
+SELECT STRING_AGG(a, char(10)) FROM string_agg_t
+GO
+~~START~~
+text
+c<newline>b<newline>g<newline>a<newline>e<newline>d<newline>h
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+b<newline>g<newline>e
+c<newline>a<newline>d<newline>h
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+h<newline>d<newline>c<newline>a
+e<newline>g<newline>b
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+h<newline>d<newline>c<newline>a
+e<newline>g<newline>b
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a<newline>c<newline>d<newline>h
+b<newline>g<newline>e
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a<newline>c<newline>h<newline>d
+g<newline>b<newline>e
+~~END~~
+
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+a<newline>c<newline>d<newline>h
+b<newline>g<newline>e
+~~END~~
+
+
+-- Batch statements
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+~~START~~
+text
+h<newline>d<newline>c<newline>a
+e<newline>g<newline>b
+~~END~~
+
+~~START~~
+text
+h<newline>d<newline>c<newline>a
+e<newline>g<newline>b
+~~END~~
+
+~~START~~
+text
+a<newline>c<newline>d<newline>h
+b<newline>g<newline>e
+~~END~~
+
+~~START~~
+text
+a<newline>c<newline>h<newline>d
+g<newline>b<newline>e
+~~END~~
+
+~~START~~
+text
+a<newline>c<newline>d<newline>h
+b<newline>g<newline>e
+~~END~~
+
+
+-- expression as column with multibyte characters
+SELECT STRING_AGG(a,'-') FROM string_agg_multibyte_t
```

Comment:
Add one or two test cases where we cast result to nvarchar so that we can verify the output

Reply:
Done